src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -698,8 +698,11 @@ export async function getFeatureIdType(params: CommonParams, userFeatureId: stri
 
 	// Legacy feature-set ID
 	if (!userFeatureId.includes('/') && !userFeatureId.includes('\\')) {
-		output.write(`Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.`, LogLevel.Error);
-		throw new ContainerError({ description: `Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements.` });
+		output.write(`Legacy feature '${userFeatureId}' not supported. Please check https://containers.dev/features for replacements. \n
```

Comment:
- Since it is twice the same message, could you extract it to a local variable?
- There is indentation on the second line of the message that needs to be removed.
- You could use an actual newline in stead of `\n`.

Reply:
@samruddhikhandale @chrmarti,
kindly check the modifications done to address this.